SU-SWS/stanford_profile
# D8CORE-933: screen reader only text for describing WYSIWYG paragraphs

# READY FOR REVIEW

# Summary
Based on input from Level Access, the visual styles in our WYSIWYG do not lend themselves to screen reader. This adds text that describes the various paragraph types when accessed by the screen reader.

# Needed By (Date)
6/9/2020

# Urgency
Not urgent

# Steps to Test

1. Check out the branch
1. Using the WYSIWYG create paragraphs for each style in the WYSIWYG
2. Using a screen reader, listen for descriptions of each paragraph

Please give feedback on the text I included. 

I would really like to have this put up for Ishi and @kerri-augenstein  to experience, and for review by Level Access.


# Affected Projects or Products
D8Core

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-933
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
